tests/framework/CodeInterfaceTests/Neutrino/Neutrino_Base/sampleNeutrino/1/results.csv

```diff
@@ -0,0 +1,302 @@
+time,result
+0.0292761,0.0266502

```

Comment:
This is generated by your interface right? 
If so, it should not be added in the repository..
Only the NEUTRINO output files (that your interface will later convert into CSVs) should be added

Reply:
I originally tried to do that, but the issue is that whenever the test is ran, it would delete the output files at the beginning of the test and then error out saying that no file exists. Is there a way around this?